src/inmanta/server/services/orchestrationservice.py

```diff
@@ -110,12 +110,11 @@ def ensure_no_cross_resource_set_dependencies(self) -> None:
         set.
         """
         for res in self.resources:
+            # It's sufficient to only check the requires relationship. The provides
+            # relationship is always a subset of the provides relationship.
```

Comment:
```suggestion
            # It's sufficient to only check the requires relationship. The provides
            # relationship is always a subset of the reverse relationship.
```
I'm not sure if this is what you're trying to say?

Reply:
Yes. Indeed. This is what I wanted to day. I will make this more clear.